src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- NuGet -->
+    <Version>1.19.3</Version>
+    <Title>Datadog APM</Title>
+    <Description>DuckType library for Datadog APM</Description>
+  </PropertyGroup>
+
+  <!-- For VS testing purposes only, copy all implementation assemblies to the
+       to test the loading of managed profiler assemblies from disk -->
+  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and ('$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1')">
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+  
+  <ItemGroup>
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1'">
+    <!--
+    This reference allows us to build the code without precompiler directives,
+    but the logic at runtime will never try to use the Registry if it's not available
+    -->
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.6.0" PrivateAssets="all" />
```

Comment:
You can probably get rid of this package reference right?

Reply:
Removed in https://github.com/DataDog/dd-trace-dotnet/pull/946/commits/e3020964dcbb2b80b94368777abbdd5b1de4e248